### ISSUE TITLE
[bug]: Recently opened in documentation is not accessible

### ISSUE THREAD

**Issue by leagrdv** [swisspost/design-system#5231]
### 🐞 Describe the bug

There is a hidden "Recently opened" section in the storybook navigation, that generates an accessibility error.
Either remove it completely from the DOM if we don't want it, or make it accessible by adding text to the delete button.

### 📷 Screenshots

What the section looks like if I remove the `display: none` to it.
![Image](https://github.com/user-attachments/assets/b0526140-5f32-461e-bc2c-8ad6117e57bd)

The delete button causes an issue as it only contains a svg file and that is not accessible.
![Image](https://github.com/user-attachments/assets/058942fd-8e76-45dd-89ae-0d544e14105a)

**Comment by gfellerph** [comment]
Quick disucssion: remove feature completely.
